### PR TITLE
Bring MultiChoice test coverage to 100%

### PR DESCRIPTION
**Pull request by gavinorland**
Also re-wrote tests to be more explicit and to no longer send in unexpected `error` prop.

When this and the [Paragraph test](https://github.com/UKHomeOffice/govuk-react/pull/396) are merged it'll bring govuk-react coverage to 100%.

**Checklist**:

* [ ] Documentation N/A
* [ ] Tests N/A
* [x] Ready to be merged

### CODE REVIEW

File: components/multi-choice/src/test.js [hunk 1]
```diff
@@ -1,38 +1,33 @@
...
-  const meta = {
-    touched: true,
-    error: example,
-  };
-  const wrapper = (
-    <MultiChoice meta={meta} hint={example} label={example}>
-      {example}
-    </MultiChoice>);
+  it('renders with a label', () => {
+    const wrapper = mount(<MultiChoice label="Label">Example</MultiChoice>);
```
Comment (penx) [inline components/multi-choice/src/test.js]: Ideally the different fixtures used by tests should be in `fixtures.js` and used by both `stories.js` and `test.js` to ensure that what we have covered by snapshot tests also appears in our visual regression tests.

Comment (penx) [reply components/multi-choice/src/test.js]: would these new snapshot tests be useful to have on storybook examples/chromatic?
If so, please add them via fixtures.js
If not, why not?

Comment (gavinorland) [reply components/multi-choice/src/test.js]: 👍 I'll take a look at this and see if I can break out anything shared.

File: components/multi-choice/src/stories.js [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
-      Yes
-    </Radio>
-    <Radio name="group1" inline>
-      No
-    </Radio>
-  </MultiChoice>
-));
+const examples = storiesOf('Form/MultiChoice/Examples', module);
+examples.addDecorator(withKnobs);
+examples.add('With a hint', () => <MultiChoiceWithKnobsHint />);
```
Comment (penx) [inline components/multi-choice/src/stories.js]: does`examples.add('With a hint', MultiChoiceWithKnobsHint);` work?

Comment (penx) [reply components/multi-choice/src/stories.js]: approved other than this

**Pull request merged**